pkg/drift/drift.go

```diff
@@ -58,21 +63,38 @@ func Process(ctx context.Context, organizationID, bucketQuery string) error {
 	if err != nil {
 		return fmt.Errorf("failed to parse IAM from Terraform State: %w", err)
 	}
-	logger.Debugw("GCP IAM entries", "number_of_entries", len(gcpIAM))
-	logger.Debugw("Terraform IAM entries", "number_of_entries", len(tfIAM))
+	logger.Debugw("gcp iam entries", "number_of_entries", len(gcpIAM))
+	logger.Debugw("terraform iam entries", "number_of_entries", len(tfIAM))
 
-	clickOpsChanges := difference(gcpIAM, tfIAM)
-	missingTerraformChanges := difference(tfIAM, gcpIAM)
+	clickOpsChanges := differenceMap(gcpIAM, tfIAM)
+	missingTerraformChanges := differenceMap(tfIAM, gcpIAM)
 
-	logger.Debugw("Found Click Ops Changes", "number_of_changes", len(clickOpsChanges))
-	logger.Debugw("Found Missing Terraform Changes", "number_of_changes", len(missingTerraformChanges))
+	ignored, err := driftignore(ctx, driftignoreFile)
+	if err != nil {
+		return fmt.Errorf("failed to parse driftignore file: %w", err)
+	}
+
+	clickOpsNoIgnoredChanges := differenceSet(clickOpsChanges, ignored)
+	missingTerraformNoIgnoredChanges := differenceSet(clickOpsChanges, ignored)
+
+	logger.Debugw("found click ops changes",
+		"number_of_changes", len(clickOpsNoIgnoredChanges),
+		"number_of_ignored_changes", len(clickOpsChanges)-len(clickOpsNoIgnoredChanges))
+	logger.Debugw("found missing terraform changes",
+		"number_of_changes", len(missingTerraformNoIgnoredChanges),
+		"number_of_ignored_changes", len(missingTerraformChanges)-len(missingTerraformNoIgnoredChanges))
 
 	// Output to stdout to mimic bash script for now.
+	// TODO(dcreey): Determine cleaner API that aligns with using the cli tool.
 	if len(clickOpsChanges) > 0 {
-		fmt.Println("Found Click Ops Changes \n>", strings.Join(clickOpsChanges, "\n> "))
+		uris := keys(clickOpsNoIgnoredChanges)
+		sort.Strings(uris)
+		fmt.Println("Found Click Ops Changes \n>", strings.Join(uris, "\n> "))
```

Comment:
Prefer `c.Outf` instead. It accomplishes the same thing, but makes it easier to test specific output.